src/NzbDrone.Core/ImportLists/Sonarr/SonarrV3Proxy.cs

```diff
@@ -90,11 +96,18 @@ private List<TResource> Execute<TResource>(string resource, SonarrSettings setti
 
             var baseUrl = settings.BaseUrl.TrimEnd('/');
 
-            var request = new HttpRequestBuilder(baseUrl).Resource(resource).Accept(HttpAccept.Json)
-                .SetHeader("X-Api-Key", settings.ApiKey).Build();
+            var request = new HttpRequestBuilder(baseUrl).Resource(resource)
+                .Accept(HttpAccept.Json)
+                .SetHeader("X-Api-Key", settings.ApiKey)
+                .Build();
 
             var response = _httpClient.Get(request);
 
+            if ((int)response.StatusCode > 299)
```

Comment:
Can we just do `response.HasHttpRedirect` here? HttpClient already throws for 400 and up given SuppressHttpError is false, right?